From e2a496c150a80964fb39f462b13474ee2ca14d81 Mon Sep 17 00:00:00 2001
From: Michal Suchanek <msuchanek@suse.de>
Date: Thu, 10 Oct 2024 19:15:00 +0200
Subject: [PATCH] cxl/json: Fix tracefs include.

../cxl/json.c:12:10: fatal error: tracefs/tracefs.h: No such file or
directory

pkg-config --cflags libtracefs
-I/usr/include/libtracefs -I/usr/include/traceevent

find /usr/include/ -name tracefs.h
/usr/include/libtracefs/tracefs.h

Signed-off-by: Michal Suchanek <msuchanek@suse.de>
---
 cxl/json.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cxl/json.c b/cxl/json.c
index 5066d3be..e65bd803 100644
--- a/cxl/json.c
+++ b/cxl/json.c
@@ -9,7 +9,7 @@
 #include <json-c/json.h>
 #include <json-c/printbuf.h>
 #include <ccan/short_types/short_types.h>
-#include <tracefs/tracefs.h>
+#include <tracefs.h>
 
 #include "filter.h"
 #include "json.h"
